test/server/cards/03_TWI/events/GuardingTheWay.spec.ts

```diff
@@ -0,0 +1,60 @@
+describe('Guarding the Way', function () {
+    integration(function (contextRef) {
+        describe('Guarding the Way\'s ability', function () {
+            it('should give a unit Sentinel and +2/+2 for the phase', function () {
```

Comment:
nit, clarity: should include "when the controller has initiative"

Reply:
does this work?

`should give a unit Sentinel for the phase and when the controller has initiative also give it +2/+2`